dist/vue-ics.common.js

```diff
@@ -184,7 +184,7 @@ var install = function install(Vue) {
       var start = start_year + start_month + start_day + start_time;
       var end = end_year + end_month + end_day + end_time;
       var now = now_year + now_month + now_day + now_time;
-      var Event = "\n    BEGIN:VEVENT\n    UID:".concat(UID, "@").concat(options.uidDomain, "\n    ").concat(url ? 'URL:' + url : '', "\n    DESCRIPTION:").concat(description).concat(rruleString ? '\n' + rruleString : '', "\n    DTSTAMP;VALUE=DATE-TIME:").concat(now, ",\n    DTSTART;VALUE=DATE-TIME:").concat(start, "\n    DTEND;VALUE=DATE-TIME:").concat(end, "\n    LOCATION:").concat(location, "\n    ").concat(organizer ? 'ORGANIZER;CN=' + organizer.name + ':MAILTO:' + organizer.email : '', "\n    SUMMARY;LANGUAGE=").concat(language, ":").concat(subject, "\n    END:VEVENT\n      ");
+      var Event = "BEGIN:VEVENT\n        UID:".concat(UID, "@").concat(options.uidDomain, "\n        ").concat(url ? 'URL:' + url : '', "\n        DESCRIPTION:").concat(description).concat(rruleString ? '\n' + rruleString : '', "\n        DTSTAMP;VALUE=DATE-TIME:").concat(now, ",\n        DTSTART;VALUE=DATE-TIME:").concat(start, "\n        DTEND;VALUE=DATE-TIME:").concat(end, "\n        LOCATION:").concat(location, "\n        ").concat(organizer ? 'ORGANIZER;CN=' + organizer.name + ':MAILTO:' + organizer.email : '', "\n        SUMMARY;LANGUAGE=").concat(language, ":").concat(subject, "\n        END:VEVENT\n      ");
```

Comment:
This is a nightmare, we should string interpolate this. (in a future change)

Reply:
this file is auto generated, so we don't have full control over this. i modified the index.js so that this file looks a little better. do you still think it's worth trying to beautify?